function.php

```diff
@@ -41,6 +41,7 @@ function displayAllCards(array $cardSets): string
     $str = "";
     foreach ($cardSets as $cardSet) {
         if (empty($cardSet)) return false;
+        if ($cardSet['deleted'] == 1) continue;
```

Comment:
Rather than filtering out deleted like this, you should actually use a where clause on your query to exclude anything where deleted = 1